client/src/actions/sftp.js

```diff
@@ -143,31 +143,25 @@ export const sftp_chmod = (fm, cwd, name, mode, recursive) => {
             fm.showAlert('Error: ' + error.message);
         }
     });
-}
+};
 
 export const sftp_quota = (fm, cwd, ms) => {
-    fm.setState({
```

Comment:
we might want to load silently because not all targets have a quota / support quota checking